src/ontology/caro-edit.owl

```diff
@@ -770,7 +771,7 @@ SubClassOf(<http://purl.obolibrary.org/obo/CARO_0001007> <http://purl.obolibrary
 # Class: <http://purl.obolibrary.org/obo/CARO_0001008> (gross anatomical part)
 
 AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/CARO_0001008> "Anatomical structure that is part of a multicellular organism and is at the gross anatomical level, e.g. above the level of a cell.  Included are portions of organism substances such as blood, multi-cell-part structures such as axon tracts, acellular anatomical structures such as hair, and organism subdivisions such as head. Excluded is the whole organism and more granular parts of the organism, such as atoms, molecules, macromolecular complexes and cells. "^^xsd:string)
-AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000117> <http://purl.obolibrary.org/obo/CARO_0001008> "mah"^^xsd:string)
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000117> <http://purl.obolibrary.org/obo/CARO_0001008> <https://orcid.org/0000-0001-9114-8737>)
```

Comment:
There is a very sad situation that MAH are two people, the second one being:

https://dblp.org/pid/33/1638.html

Reply:
You will have to check with @dosumis on slack (he wont answer here) if we can assume for all CARO ids that MAH is melissa, and not midori (both have curated thousands of classes)

Reply:
this is a mega facepalm.

Reply:
MAH in CARO is all Melissa.

Reply:
that's what I assumed since she was on the ontology-level authors list. Thanks for confirming